libraries/AP_HAL_ChibiOS/UARTDriver.cpp

```diff
@@ -1834,4 +1834,15 @@ void usb_initialise(void)
 }
 #endif
 
+// disable TX/RX pins for unusued uart
+void UARTDriver::disable_rxtx(void) const
+{
+    if (arx_line) {
+        palSetLineMode(arx_line, PAL_MODE_INPUT);
```

Comment:
Not PAL_MODE_UNCONNECTED?

Reply:
UNCONNNECTED applies a pullup: ```#define PAL_MODE_UNCONNECTED            PAL_MODE_INPUT_PULLUP``` 
I didn't want the pullup